release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+
+{product-title} {product-version} clusters are available at
+https://cloud.openshift.com/openshift. The Red Hat OpenShift Cluster Manager
+application for {product-title} allows you to deploy OpenShift clusters to
+either on-premise or cloud environments.
+
+{product-title} {product-version} is supported on Red Hat Enterprise Linux 7.6 and
+later, as well as Red Hat Enterprise Linux CoreOS 4.1.
+
+You must use {op-system-first} for the control plane, or master, machines and
+can use either {op-system} or Red Hat Enterprise Linux 7.6 for compute, or worker,
+machines.
+
+[IMPORTANT]
+====
+Because only Red Hat Enterprise Linux version 7.6 is supported for compute
+machines, you must not upgrade the Red Hat Enterprise Linux compute machines to
+version 8.
+====
+
+[id="ocp-4-2-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[id="ocp-4-2-installation-and-upgrade"]
+=== Installation and upgrade
+
+[id="ocp-4-2-openshift-install"]
+==== CLI-based installation
+
+{product-title} {product-version} introduces a new CLI-based installer called
+`openshift-install`. It is designed to simplify provisioning OpenShift on
+immutable installer provisioned infrastructure using an interactive guided
+workflow within about 30 minutes. This approach provides an OpenShift deployment
+with full stack automation of host OS and platform updates and infrastructure
+management without the complexity of having to provision your own
+infrastructure. Only minimal user input is needed with all non-essential
+installation configuration options, which are now configured post-installation
+via component operator Custom Resource Definitions (CRDs).
+
+Updates are performed largely the same way. OpenShift update content must be
+mirrored first to the local container registry first then the administrator
+tells `oc adm upgrade` where to pull the update content from.
+
+[id="ocp-4-2-disconnected-install"]
+==== Disconnected installation
+
+{product-title} {product-version} introduces support for installation and
+updating {product-title} clusters in disconnected, restricted network
+installation environments. Its designed to work with any docker 2.2
+specification complaint container registry for hosting OpenShift content.
+Administrators first need replicate content from Quay.io to their local
+container registry. Once that is done, `openshift-install` can be configured to
+generate ignition configurations that pull content locally rather than from
+Quay.io. This is designed to work with user-provisioned infrastructure
+deployments only.
+
+For disconnected environments, you are still able to use Operators from the OLM
+catalog. However, you must manually pull the Operator sources in order to
+populate the the offline catalog. This manual process is only a temporary
+workaround for {product-title}. 4.2. An automated solution is coming in a future
+release.
```

Comment:
@gabemontero do we have a note elsewhere about the restrictions on builds in disconnected envs?  (sounds like the restriction is "they won't work" at this point since there is no way for the user to get the creds to be correct for the mirror)

Reply:
nm, found the note/limitation at the bottom.